sycl/test-e2e/Graph/RecordReplay/work_group_size_prop.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %{build} -o %t.out
+// RUN: %{build} -o %t.out -Wno-deprecated-declarations
```

Comment:
I'd think we'd need this on the `Graph/Explicit/work_group_size_prop.cpp` test as well

Reply:
Yes I think you're correct as it's basically the same as Graph/RecordReplay/work_group_size_prop.cpp. Fixed!